Makefile

```diff
@@ -6,6 +8,14 @@ CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
 OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
 KUBECTL ?= kubectl
 
+# for boilerplate
+OPERATOR_NAME=route-monitor-operator
+MAINPACKAGE=.
+TESTTARGETS=$(shell ${GOENV} go list -e ./... | egrep -v "/(vendor)/" | grep -v /int)
+
+# need to override boilerplate targets which are not working on this operator
+op-generate openapi-generate: ;
+
 VERSION ?= $(OPERATOR_VERSION)
 PREV_VERSION ?= $(VERSION)
```

Comment:
L33 (which I can't comment on directly): `IMG` is [set in standard.mk](https://github.com/openshift/route-monitor-operator/blob/97404eca7e5b669d904607c6a357d459fecb9116/boilerplate/openshift/golang-osd-operator/standard.mk#L28) as well. Since L33 comes after the `include`, but uses `?=`, it will be a no-op.

AFAICS this will break the [`pre-deploy`](https://github.com/openshift/route-monitor-operator/pull/82/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R74-R75) and [`podman-*`](https://github.com/openshift/route-monitor-operator/pull/82/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R105-R111) targets.

Reply:
`pre-deploy` I guess will be called in most cases with a externally-set IMG variable so we should be fine removing the line.